src/App.js

```diff
@@ -7,8 +7,10 @@ import Home from "./pages/home/Home";
 import Printer from "./pages/printer/Printer";
 import Search from "./pages/search/Search";
 import "./App.css";
+import { useState } from "react";
 
 function App() {
+  const [showTheme, setShowTheme] = useState(true);
```

Comment:
@nitink-prog Could you make sure the default state is false because it should be collapse when the user landed first time on this page.

Reply:
Yes, this is in the new commit.